sign_node/signer.py

```diff
@@ -114,61 +132,131 @@ def on_sync_request(self, queue, message):
             )
             sig_type = defaultdict(
                 lambda: '--detach-sign',
-                **{'clear-sign': '--clear-sign',
-                   'detach-sign': '--detach-sign'})
+                **{
+                    'clear-sign': '--clear-sign',
+                    'detach-sign': '--detach-sign',
+                }
+            )
             with tempfile.NamedTemporaryFile(mode='w') as fd:
-                asc_file_name = f'{fd.name}.asc'
+                asc_file_name = Path(fd.name).with_suffix('.asc')
```

Comment:
you can declare suffix `.asc` in `tempfile.NamedTemporaryFile`
```python3
with tempfile.NamedTemporaryFile(mode='w', suffix='.asc') as fd:
    ...
```

Reply:
Thanks!